web/packages/teleterm/src/main.ts

```diff
@@ -194,6 +195,29 @@ async function initializeApp(): Promise<void> {
   });
 }
 
+/**
+ * There is an outstanding issue about Electron storing its caches in the wrong location https://github.com/electron/electron/issues/8124.
+ * Based on the Apple Developer docs and the discussion under that issue, changing the location of `sessionData`
```

Comment:
I'd add a link to those Apple docs as well.

Reply:
Added.